src/Chainweb/Mempool/InMem.hs

```diff
@@ -162,10 +161,10 @@ withInMemoryMempool_ :: Logger logger
                      -> ChainwebVersion
                      -> (MempoolBackend t -> IO a)
                      -> IO a
-withInMemoryMempool_ l cfg v f = do
+withInMemoryMempool_ l cfg _v f = do
```

Comment:
why not just remove the version from the signature? Same for below

Reply:
This is part of the main mempool API, which I did't want to change. It also seems reasonable that the mempool is aware of the chainweb version, even if the current implementation doesn't use it.